docs/download_files.md

```diff
@@ -2,6 +2,7 @@
 
 The ScPCA Portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
 These files are delivered as a zip file.
+When you uncompress the zip file, the root directory name of your download will include the date you accessed the data on the ScPCA Portal.
```

Comment:
```suggestion
When you uncompress the zip file, the root directory name of your download will include the date you accessed the data on the ScPCA Portal.
It is important to make note of this date if you wish to publish or present the data or your results in the future.
See [How to Cite](link) section for more information
```
Adding why we give them the accessed date. Is it too much spoon-feeding?
 It's a suggestion, I'm okay if we decide to not include it.

Reply:
What about?

```suggestion
When you uncompress the zip file, the root directory name of your download will include the date you accessed the data on the ScPCA Portal.
We recommend you record this date in case there are future updates to the Portal that change the underlying data or if you need to cite the data in the future (see [How to Cite](TODO: LINK) for more information).
<!-- TODO: Add link to CHANGELOG -->
```

Reply:
Yes, this looks good!